### PR TITLE
Remove .0 from macos builder name

### PR DESCRIPTION
**Pull request by clintoncwolfe**
Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New content (non-breaking change)
- [ ] Breaking change (a content change which would break existing functionality or processes)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Review by kagarmoe — APPROVED** [review]

File: .expeditor/release.omnibus.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ builder-to-testers-map:
   mac_os_x-10.13-x86_64:
```
Comment (tduffield) [inline .expeditor/release.omnibus.yml:25]: You'll also want to switch your builder from 10.13 to 10.14. We now support building on 10.14 and have deprecated 10.13.

**Review by tduffield — CHANGES_REQUESTED** [review]
There is a bug with this logic upstream. Need to sort that out before this is merged.

**Comment by tduffield** [comment]
@clintoncwolfe I'd like to get the 10.14 builder change out, so we should either open a new PR with just that change, or update this PR to keep the `11.0` and merge it.

**Comment by clintoncwolfe** [comment]
I've moved the change to 10.14 onto a separate PR, #5415

**Review by tduffield — APPROVED** [review]
Alright, we are ready to merge this :) Thanks for your patience.

**Pull request merged**